sinks/librato.py

```diff
@@ -288,13 +285,16 @@ def add_measure(self, key, value, time):
             k = name
```

Comment:
You should also probably avoid setting `tags["source"]` if the key already exists in the tags.  I've updated our statsd client to put "source" into tags for legacy code, and using a recognizable fake prefix on metric names in order to smoothly transition from using a metric prefix source altogether.